java/src/jmri/jmrix/ActiveSystemsMenu.java

```diff
@@ -51,9 +51,9 @@ static public void addItems(JMenuBar m) {
             }
         }
 
-        if (jmri.jmrix.easydcc.ActiveFlag.isActive()) {
-            m.add(getMenu("jmri.jmrix.easydcc.EasyDCCMenu"));
-        }
+//        if (jmri.jmrix.easydcc.ActiveFlag.isActive()) { // migrated 2017 4.9.5
+//            m.add(getMenu("jmri.jmrix.easydcc.EasyDccMenu"));
+//        }
```

Comment:
Don't just comment this line out, but remove it

Reply:
done